src/applications/check-in/hooks/useDatadogRum.jsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useMemo } from 'react';
+import { datadogRum } from '@datadog/browser-rum';
+
+import environment from 'platform/utilities/environment';
+import { useSelector } from 'react-redux';
+import { makeSelectFeatureToggles } from '../utils/selectors/feature-toggles';
+
+const initializeDatadogRum = () => {
+  if (
+    // Prevent RUM from running on local/CI environments.
+    environment.BASE_URL.indexOf('localhost') < 0 &&
+    // Prevent re-initializing the SDK.
+    !window.DD_RUM?.getInitConfiguration()
+  ) {
+    datadogRum.init({
+      applicationId: 'f61cc984-0ba6-4502-b4d0-fcae00cb8422',
+      clientToken: 'pub65530b937303a0163a4ab3320e818340',
```

Comment:
are these ok for a public repo?

Reply:
Great question, discussed with platform [on slack](https://dsva.slack.com/archives/CBU0KDSB1/p1666375988481699).